task19/css/style.css

```diff
@@ -0,0 +1,235 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+.loader,
+.loader:before,
+.loader:after {
+  background: rgb(8, 2, 58);
+  -webkit-animation: load1 1s infinite ease-in-out;
+  animation: load1 1s infinite ease-in-out;
+  width: 1em;
+  height: 4em;
+}
+.loader {
+  color: rgb(8, 2, 58);
+  text-indent: -9999em;
+  margin: 88px auto;
+  position: relative;
+  font-size: 11px;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation-delay: -0.16s;
+  animation-delay: -0.16s;
+}
+.loader:before,
+.loader:after {
+  position: absolute;
+  top: 0;
+  content: "";
+}
+.loader:before {
+  left: -1.5em;
+  -webkit-animation-delay: -0.32s;
+  animation-delay: -0.32s;
+}
+.loader:after {
+  left: 1.5em;
+}
+@-webkit-keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+@keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+
+
+
+/* slider */
+
+.slide-image-item {
+  z-index: 1;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 300px;
+  padding-top: 30px;
+  display: none;
+}
+.slide-image-item.is-active {
+  z-index: 2;
+  text-align: center;
+  display: block;
+}
+.slide-image {
+  max-width: 300px;
+}
+
+.slide-button {
+  border: none;
+  z-index: 10;
+  width: auto;
+  height: 30px;
+  overflow: visible;
+  cursor: pointer;
+}
+
+.slide-button:hover,
+:disabled {
+  opacity: 0.7;
+}
```

Comment:
IMO
When disabled, I thought it would be better to use` cursor: default` or `cursor: not-allowed` to let the user know that it cannot be pressed.

Reply:
I should write the style from the user's perspective. Thank you very much.
[e31df28](https://github.com/seeeen044/JavaScript_handson/pull/20/commits/e31df2890eba934b08d2e1c0457e4a2311bf14fd)